docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md

```diff
@@ -0,0 +1,736 @@
+      "type": "string",
+      "configurable": false,
+      "private": true
+    },
+    "confirmed": {
+      "type": "boolean",
+      "default": false,
+      "configurable": false
+    },
+    "blocked": {
+      "type": "boolean",
+      "default": false,
+      "configurable": false
+    },
+    "role": {
+      "model": "role",
+      "via": "users",
+      "plugin": "users-permissions",
+      "configurable": false
+    },
+    "products": {
+      "collection": "product",
+      "via": "owner"
+    }
+  }
+}
+```
+
+#### Admin
+
+For now you won't be able to customize a plugin's admin. This feature will come soon so keep an eye out for this feature.
+
+### Migrating local plugins
+
+If you have local plugins (plugins in the `./plugins` that don't exist on npm) you can leave them in the plugins folder.
+
+The only difference is that the admin of a local plugin is ignored for the moment.
+
+## Migrating `api`
+
+### Migrating controllers and services
+
+In the `beta` we are introducing the `Core API`, which is replacing the templates that were generated before.
+
+Now when you create a new model your `controller` and `service` will be empty modules and will be used to override the default behaviours.
+Read more about [controllers](https://strapi.io/documentation/3.0.0-beta.x/guides/controllers.html) or [services](https://strapi.io/documentation/3.0.0-beta.x/guides/services.html)
+
+To migrate, you will only have to delete the methods you haven't modified or created from your `controllers` and `services`
+
+**Before**
+
+`./api/product/controllers/Product.js`
+
+```js
+module.exports = {
+  find: async (ctx) => {
+	  myCustomImplementation(ctx);
+  },
+
+  customAction: async(ctx) => {
+    ctx.send({ ok: true });
+  }
+
+  findOne: async (ctx) => {
+    return strapi.services.product.fetch(ctx.params);
+  },
+
+  count: async (ctx) => {
+    return strapi.services.product.count(ctx.query);
+  },
+
+  create: async (ctx) => {
+    return strapi.services.product.add(ctx.request.body);
+  },
+
+  update: async (ctx) => {
+    return strapi.services.product.edit(ctx.params, ctx.request.body) ;
+  },
+
+  destroy: async (ctx) => {
+    return strapi.services.product.remove(ctx.params);
+  }
+};
+
+```
+
+**After**
+
+`./api/product/controllers/Product.js`
+
+```js
+module.exports = {
+  // this function overrides the default one
+  find: async ctx => {
+    myCustomImplementation(ctx);
+  },
+
+  // this one is added to the default controller
+  customAction: async ctx => {
+    ctx.send({ ok: true });
+  },
+};
+```
+
+> The same goes for services
+
+For custom `controllers` and `services` (the ones without a model) you can leave them untouched.
+
+### Migrating routes
+
+In the file `./api/apiName/config/routes.json` we renamed the `destroy` action to `delete` you will have to update your `routes` and `controller` accordingly.
+
+If you haven't customized the `destroy` action then remove it from your controller instead of renaming it.
```

Comment:
If you haven't customized the `destroy` action, then remove it from your controller **rather than** renaming it.